screenfetch-dev

```diff
@@ -991,7 +991,7 @@ detectcpu () {
 			cpu="Unknown"
 		fi
 	elif [[ "$distro" == "FreeBSD" || "$distro" == "DragonflyBSD" ]]; then
-		cpu=$(sysctl -n hw.model)
+		cpu=$(dmesg | grep CPU: | head -n 1 | sed -r "s/CPU: //" | sed -e 's/([^()]*)//g')
```

Comment:
Ran the intel test:

$ sysctl -n hw.model
Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
$ dmesg | grep CPU: | head -n 1 | sed -r "s/CPU: //" | sed -e 's/([^()]*)//g'
Intel Core i7-4870HQ CPU @ 2.50GHz